runners/reference/job-server/build.gradle

```diff
@@ -22,13 +22,17 @@ applyJavaNature()
 description = "Apache Beam :: Runners :: Reference :: Job Orchestrator"
 
 dependencies {
+  compile project(path: ":beam-runners-parent:beam-java-fn-execution", configuration: "shadow")
```

Comment:
ditto on shadow "scope" here and below on the other compile you added

Reply:
Done.